data/mnc_job.json

```diff
@@ -0,0 +1,157 @@
+        "index": 9,
+        "image": "   _______     \n  /       \\    \n |   ____  |   \n |   |    | |   \n |   |    | |   \n |   |    | |   \n  \\_______/     \n      | |       \n     _|_|_      \n    /     \\     \n   /       \\    ",
+        "question": "Your company is hosting a weekend team-building retreat. Do you attend?",
+        "options": [
+          { "text": "Go enthusiastically to network and bond with colleagues.", "points": { "money": 0, "health": -5, "happiness": 10 } },
+          { "text": "Skip it for some much-needed personal time.", "points": { "money": 0, "health": 5, "happiness": 5 } },
+          { "text": "Attend only the key sessions to show your presence.", "points": { "money": 0, "health": 0, "happiness": 5 } }
+        ]
+      },
+      {
+        "index": 10,
+        "image": "   ________      \n  /        \\     \n |    MNC   |    \n |    ___   |    \n |   /   \\  |    \n |  |     | |    \n  \\_________/     \n      | |        \n     _|_|_       \n    /     \\      \n   /       \\     ",
+        "question": "A promotion opportunity arises, but it requires frequent travel. What’s your choice?",
+        "options": [
+          { "text": "Accept the role, excited about new challenges.", "points": { "money": 15, "health": -5, "happiness": -10 } },
+          { "text": "Decline and wait for a better-suited promotion.", "points": { "money": 0, "health": 5, "happiness": 5 } },
+          { "text": "Negotiate for limited travel to balance work-life.", "points": { "money": 10, "health": -5, "happiness": 10 } }
+        ]
+      },
+      {
+        "index": 11,
+        "image": "    ________     \n   /        \\    \n  |   O O   |    \n  |    __    |   \n  |   /  \\   |   \n   \\_________/    \n      | |        \n      | |        \n      |_|        \n    ___|_|___    \n   /        \\   ",
+        "question": "Your boss offers you a chance to lead a high-stakes project. How do you react?",
+        "options": [
+          { "text": "Take the opportunity and work overtime to succeed.", "points": { "money": 15, "health": -10, "happiness": -10 } },
+          { "text": "Suggest someone more experienced for the role.", "points": { "money": 0, "health": 10, "happiness": 10 } },
+          { "text": "Offer to co-lead with a colleague to share the load.", "points": { "money": 10, "health": -5, "happiness": 5 } }
+        ]
+      },
+            {
+                "index": 12,
+                "image": "    ____       ____       ____\n   /    \\     /    \\     /    \\\n  |      |   |      |   |      |\n  |______|   |______|   |______|\n   |    |     |    |     |    |\n   |    |     |    |     |    |\n  __|____|____|____|____|____|__\n | | |  | | |  | | |  | |  | |",
+                "question": "Your team is underperforming. How do you address it?",
+                "options": [
+                    { "text": "Set up one-on-one meetings to understand issues.", "points": { "health": 0, "happiness": 15, "wealth": 0 } },
```

Comment:
Here it is correct. Cross check everywhere. Not adding any more comments.

Reply:
corrected it.